Source/Core/Core/HW/VideoInterface.cpp

```diff
@@ -661,6 +665,10 @@ static void EndField()
 // Run when: When a frame is scanned (progressive/interlace)
 void Update()
 {
+	if (s_half_line_of_next_si_poll == s_half_line_count) {
```

Comment:
Curly braces are a bit inconsistent inside this function, could you normalize them to conform to our coding guidelines (brace on next line)?